pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -24,6 +24,7 @@ import (
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/labels"
 	"k8s.io/kubernetes/pkg/util"
+	kutil "k8s.io/kubernetes/pkg/util"
```

Comment:
actually I need to get rid of kutil.StringList (see:https://github.com/openshift/origin/pull/5746 )